make/deps.mk

```diff
@@ -46,6 +47,7 @@ ifeq ($(USER),travis)
 endif
 
 dep_amqp_client_commit = rabbitmq_v3_6_0
+dep_apns = git https://github.com/inaka/apns4erl.git 2.2.0
```

Comment:
Have you tried specifying this as an hex.pm package? https://hex.pm/packages/apns4erl
I'm guessing it wouldn't work due to the app having a different name than the package but I haven't tried it.